lib/generators/shopify_app/install/templates/shopify_app.rb.tt

```diff
@@ -38,7 +38,7 @@ Rails.application.config.after_initialize do
       api_key: ShopifyApp.configuration.api_key,
       api_secret_key: ShopifyApp.configuration.secret,
       api_version: ShopifyApp.configuration.api_version,
-      host_name: URI(ENV.fetch('HOST', '')).host || '',
+      host: ENV['HOST'],
```

Comment:
Either this or https://github.com/Shopify/shopify-api-ruby/pull/1017 is incorrect.  `host_name` is still required by the `setup` function. I'm not sure if it should still be required or should be optional now. If it should be optional maybe there should be some error checking to ensure that host and host_name don't conflict?